allure-report-face/src/components/widget-overview/OverviewWidget.js

```diff
@@ -1,7 +1,53 @@
 import './styles.css';
 import {ItemView} from 'backbone.marionette';
 import template from './OverviewWidget.hbs';
+import c3 from 'c3';
```

Comment:
It must be done before merge. You are going to add 360K (in minimised version) to report size. It is 33% of current size, it is unacceptable.